server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKAuthenticator.java

```diff
@@ -116,6 +133,31 @@ public void createUser(String principal, AuthenticationToken token)
     }
   }
 
+  /**
+   * Creates user with outdated password hash for testing
+   *
+   * @deprecated since 2.1.0, only present for testing DO NOT USE!
+   */
```

Comment:
could we log a WARN message that this method has been used? that way it would show up in operator logs should we mistakenly use it in a non-test context.

Reply:
yeah that sounds great.

Reply:
Just realized the only reason for this method was that the IT tests required it, but with reworking those to unit tests this is no longer used. So I'll just remove this.